recipes/libaec/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibaecConan(ConanFile):
+    name = "libaec"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gitlab.dkrz.de/k202009/libaec"
+    description = "Adaptive Entropy Coding library"
+    topics = ("conan", "dsp", "libaec", "encoding", "decoding")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure(build_folder=self._build_subfolder)
+        return cmake
+
+    def build(self):
```

Comment:
```suggestion
    def build(self):
        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "add_subdirectory(tests)", "")
```

Avoid building the tests